src/components/cats/index.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import adoptedCats from 'redux/reducers/cats'
```

Comment:
We don't need to import here, as we are passing `adoptedCats` through props. 

Essentially what you are doing here is importing the whole reducer function from `redux/reducers/cats`